client/src/app/mainwindow.cpp

```diff
@@ -257,6 +257,7 @@ void MainWindow::slotResourceSelectionChanged(int index)
     }
     AgentInstance agent = mResourceSelector->itemData(index, AgentInstanceModel::InstanceRole).value<AgentInstance>();
     if (agent.isValid()) {
+        ClientSettings::self()->setDefaultResourceId(agent.identifier());
```

Comment:
isn't this called when selecting another resource in the combobox? We don't want to save that.

Reply:
fixed

Reply:
After using this for a while, I changed my mind. If the user switches to another resource in the combobox, it's annoying if that doesn't get remembered for next time. I'll make the change, just wanted to inform you that I'm inconsistent with myself :-)